### PR TITLE
Mitigate issues found by Component Governance

### PR DESCRIPTION
**Pull request by heaths**

### CODE REVIEW

**Comment by heaths** [comment]
These issues were found against the latest Component Governance run in Azure DevOps. Where possible, only the latest patch version was applied as close to the source of the problem as possible. The updates in eng/Packages.Data.props should take care of most issues found related to Microsoft.AspNetCore.*.

The's only one outstanding I have an internal conversation with @allenjzhang about for track 1 management SDKs.

**Review by jsquire — APPROVED** [review]

File: sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Scenario.Tests/tests/Microsoft.Azure.WebJobs.Extensions.Storage.Scenario.Tests.csproj [hunk 1]
```diff
@@ -5,8 +5,9 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" VersionOverride="2.14.0" />
-    <PackageReference Include="Microsoft.AspNetCore" VersionOverride="2.1.0" />
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" VersionOverride="2.1.0" />
+    <PackageReference Include="Microsoft.AspNetCore" VersionOverride="2.1.7" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" VersionOverride="2.1.3" />
+    <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel.Core" VersionOverride="2.1.25" />
```
Comment (kasobol-msft) [inline sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Scenario.Tests/tests/Microsoft.Azure.WebJobs.Extensions.Storage.Scenario.Tests.csproj:10]: Why is Kestrel being added here?

Comment (heaths) [reply sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Scenario.Tests/tests/Microsoft.Azure.WebJobs.Extensions.Storage.Scenario.Tests.csproj:10]: In this particular case, it's upgrading an indirect dependency. Version 2.1.0 has a CVE, and this was the most recent 2.1.x. I did update Microsoft.AspNetCore.* globally for most other projects, but those were already using 2.2.x. For security updates, we do the same as .NET: update to the latest patch version, if available.

If you're the owner of this project and are fine with updating to 2.2.x I can do so instead.

Comment (kasobol-msft) [reply sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Scenario.Tests/tests/Microsoft.Azure.WebJobs.Extensions.Storage.Scenario.Tests.csproj:10]: If tests pass then 2.2.x should be fine. let's try that.

Comment (heaths) [reply sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Scenario.Tests/tests/Microsoft.Azure.WebJobs.Extensions.Storage.Scenario.Tests.csproj]: I updated the references. If tests pass I'll merge.

**Review by kasobol-msft — APPROVED** [review]

**Pull request merged**